### PR TITLE
[Issue #1544]: Fixed composer.lock file

### PR DESCRIPTION
**Pull request by afi13**
Updated composer.lock file.
drupal/video module was updated to the latest security release, see links below for details:
https://www.drupal.org/project/video/releases/8.x-1.4
https://www.drupal.org/sa-core-2019-003

## Steps for review

- [ ] Make sure `composer.lock` has correct versions for all packages.
- [ ] Make sure you can install any new package with composer without any errors.
- [ ] Make sure `video` module is upgraded to latest security release https://www.drupal.org/project/video/releases/8.x-1.4.
- [ ] There no any regression on Open Y functionality.

## General checks
- [ ] All coding styles are fulfilled and there are no any issues reported by CodeSniffer CI. <br/>CI code sniffer errors
- [ ] All tests are running and there are no failed tests reported by CI. <br/>Behat test results
- [ ] [Documentation](https://github.com/ymcatwincities/openy/tree/8.x-1.x/docs) has been updated according to PR changes.
- [ ] [Steps for review](https://github.com/ymcatwincities/openy/pull/94#issue-204580200) have been provided according to PR changes. <br/>Steps for review
- [ ] Make sure you've provided all necessary hook\_update\_N to [support upgrade path](https://github.com/ymcatwincities/openy/blob/8.x-1.x/docs/Development/Upgrade%20path.md).
- [ ] Make sure your git email is associated with account on drupal.org, otherwise you won't get commits there. <br/>drupal.org email
- [ ] If you would like to get credits on drupal.org, [check documentation](https://github.com/ymcatwincities/openy/blob/8.x-1.x/docs/Development/Contributing.md#drupalorg-credits).

Thank you for your contribution!

### CODE REVIEW

**Review by hamrant — CHANGES_REQUESTED** [review]
@afi13 Please add to development documentation small instruction how to update contrib in composer:
https://github.com/ymcatwincities/openy/tree/8.x-2.x/docs/Development

**Review by ddrozdik — DISMISSED** [review]

**Comment by bjwilkins** [comment]
Build comment file: 
--------------

 
| Environment | Link |
| ------------- | ------------- |
| Fresh OpenY installation - Rose theme | http://openy.cibox.tools/build1551 |
| Fresh OpenY installation - Lily theme | http://lily.openy.cibox.tools/build1551 |
| Upgraded(upgrade path) installation | http://upgrade.openy.cibox.tools/build1551 |
| Installation process | http://install.openy.cibox.tools/build1551/install.php |

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://openy.cibox.tools:8080/job/PR_BUILDER_COMPOSER/1551/

**Review by hamrant — DISMISSED** [review]

File: composer.json [hunk 1]
```diff
@@ -239,15 +239,15 @@
...
+        "drupal/video": "^1.4",
         "drupal/easy_breadcrumb": "^1.8",
         "drupal/ckeditor_bootstrap_buttons": "^1.1",
         "drupal/fontyourface": "^3.2",
         "drupal/css_editor": "^1.2",
         "drupal/domain": "1.0-alpha14",
         "drupal/domain_theme_switch": "^1.4",
         "drupal/social_feed_fetcher": "^1.0",
-        "symfony/dom-crawler": "~2.8|~3.0|~4.3",
+        "symfony/dom-crawler": "~4.2",
```
Comment (podarok) [inline composer.json]: Keep old version, please
Needed for support of step by step upgrade path

**Review by podarok — CHANGES_REQUESTED** [review]

File: composer.json (hunk 1 above)
Comment (afi13) [reply composer.json]: Done

**Comment by bjwilkins** [comment]
Build comment file: 
--------------

 
| Environment | Link |
| ------------- | ------------- |
| Fresh OpenY installation - Rose theme | http://openy.cibox.tools/build1552 |
| Fresh OpenY installation - Lily theme | http://lily.openy.cibox.tools/build1552 |
| Upgraded(upgrade path) installation | http://upgrade.openy.cibox.tools/build1552 |
| Installation process | http://install.openy.cibox.tools/build1552/install.php |

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://openy.cibox.tools:8080/job/PR_BUILDER_COMPOSER/1552/

**Review by hamrant — APPROVED** [review]

**Comment by bjwilkins** [comment]
Build comment file: 
--------------

 
| Environment | Link |
| ------------- | ------------- |
| Fresh OpenY installation - Rose theme | http://openy.cibox.tools/build1553 |
| Fresh OpenY installation - Lily theme | http://lily.openy.cibox.tools/build1553 |
| Upgraded(upgrade path) installation | http://upgrade.openy.cibox.tools/build1553 |
| Installation process | http://install.openy.cibox.tools/build1553/install.php |

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://openy.cibox.tools:8080/job/PR_BUILDER_COMPOSER/1553/

**Comment by bjwilkins** [comment]
Build comment file: 
--------------

 
| Environment | Link |
| ------------- | ------------- |
| Fresh OpenY installation - Rose theme | http://openy.cibox.tools/build1554 |
| Fresh OpenY installation - Lily theme | http://lily.openy.cibox.tools/build1554 |
| Upgraded(upgrade path) installation | http://upgrade.openy.cibox.tools/build1554 |
| Installation process | http://install.openy.cibox.tools/build1554/install.php |

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://openy.cibox.tools:8080/job/PR_BUILDER_COMPOSER/1554/

**Pull request merged**